policy/sample_points/lsst/img_quality_weight.yaml

```diff
@@ -34,7 +34,3 @@
 "28": 0.2369
 "29": 0.2369
 "30": 0.2369
-"31": 0.25
```

Comment:
What do you think about keeping these in here and instead removing the code in `ts_imsim` that adds them in?

Reply:
Because right now it happens in a simulation step and I think I'd rather have OFC handle it for the real data.

Reply:
Also maybe we set the values to be 1.0 since they will be normalized anyway.